package.json

```diff
@@ -44,6 +45,10 @@
     "yargs": "^3.9.0"
   },
   "scripts": {
+    "preupdate-webdriver": "npm install",
+    "update-webdriver": "webdriver-manager update",
+    "pree2e": "npm run update-webdriver",
+    "e2e": "protractor protractor.conf.js",
```

Comment:
I think that we should move this to gulpfile, since it have all needed info about server host and port.